### PR TITLE
Testing Pull Request

### PR DESCRIPTION
**Pull request by bitoiu**
This change only affects a REAME.

@MadeHube, @vcruzdelvalle @Ksprbgh  what do you think of these changes?

Would they be good enough to be merged into master?

:octocat: :heart:

### CODE REVIEW

**Comment by vcruzdelvalle** [comment]
@bitoiu yes (y)

**Comment by Ksprbgh** [comment]
There is a scary mutant cat on there. 

Perhaps remove it to not scare me when I look at it?

**Comment by MadeHube** [comment]
Sure !

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 *Bold*
 
 @nigelabbott
+
+teste1
```
Comment (bitoiu) [inline README.md]: why was this added here? teste not even correct in English.
